corehq/apps/auditcare/tests/test_models.py

```diff
@@ -0,0 +1,20 @@
+from django.test import RequestFactory, TestCase
+from testil import Config
+
+from corehq.apps.auditcare.models import NavigationEventAudit
+
+from .test_middleware import make_view
+
+
+class TestNavigationEventAudit(TestCase):
+
+    def setUp(self):
+        self.request = RequestFactory().get("/path", {"key": "value"})
+        self.request.session = Config(session_key="abc")
+
+    def test_audit_view_should_not_save(self):
+        view = make_view()
+        event = NavigationEventAudit.audit_view(self.request, "username", view, {})
+        if event._id is not None:
+            self.addCleanup(event.delete)
```

Comment:
Does changing the class to `SimpleTestCase` make a faulty save throw an error before it can ever write? If so, can we remove this cleanup?

Reply:
Yes, that's a good point. Will change.